src/surfing_penguin/db_interface/survey_functions.py

```diff
@@ -21,7 +22,12 @@ def id_get_survey(ID):
 
 
 def name_get_survey(name):
-    return session.query(Survey).filter_by(surveyname=name).first()
+    return session.query(Survey).filter_by(surveyname=name).all()
+
+
+def author_get_survey(author):
```

Comment:
Same here, if the result is a list of surveys, please add an s after the survey (i.e., 'author_get_surveys'

Reply:
fixed